modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportOMIA.pm

```diff
@@ -209,6 +209,7 @@ sub split_omia {
 
   my $prefix = 'omia_';
   my $suffix = '.txt';
+  make_path($workdir."/omia_split") or die "Failed to create $workdir/omia_split $!\n";
```

Comment:
Should this check that the directory exists as in other updates?

Reply:
It's was not necessary, the `split_omia` method is called only once when the folder does not exist, checked at L152.